third_party/BUILD.jsoncpp

```diff
@@ -0,0 +1,107 @@
+licenses(["unencumbered"])
```

Comment:
Can you add documentation for what this does? Either here or in `startupos_external_jsoncpp` in `WORKSPACE`.

Reply:
Done.